kubectl-fdb/cmd/version.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * version.go
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2020 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+)
+
+// TODO: should follow the operator versioning
+var pluginVersion string = "0.0.1"
```

Comment:
Should we make this match the current version, and then add a note to the release guide to update the version here? We're already updating the version by find-and-replace.